cumulusci/tasks/salesforce/sourcetracking.py

```diff
@@ -133,13 +134,15 @@ def _store_snapshot(self, changes):
             json.dump(self._snapshot, f)
 
     def _reset_sfdx_snapshot(self):
-        sfdx(
-            "force:source:tracking:reset",
-            args=["-p"],
-            username=self.org_config.username,
-            capture_output=True,
-            check_return=True,
-        )
+        # If org is from sfdx, reset sfdx source tracking
+        if isinstance(self.org_config, ScratchOrgConfig):
```

Comment:
If a user `cci org connect`'s someone else's scratch org, is that a `ScratchOrgConfig`?

Reply:
No. And that is one of the cases I'm trying to avoid here, since sfdx would not know about the org and throws an error `No AuthInfo found for name [username]`